app/models/attachment_data.rb

```diff
@@ -61,9 +61,8 @@ def indexable?
 
   def all_asset_variants_uploaded?
     asset_variants = assets.map(&:variant).map(&:to_sym)
-    required_variants = pdf? ? AttachmentUploader.versions.keys.push(:original) : [Asset.variants[:original].to_sym]
 
-    (required_variants - asset_variants).empty?
+    (%i[original] - asset_variants).empty?
```

Comment:
Could still use the `Asset.variants[:original].to_sym` syntax. Alternatively, keeping the`AttachmentUploader.versions.keys.push(:original)` logic here future-proofs us from when we might reintroduce a variant to that uploader.

Reply:
> Could still use the `Asset.variants[:original].to_sym` syntax

Could do, but that seems unnecessarily cryptic. We've started just using the `:original` symbol directly [in other places in Whitehall](https://github.com/alphagov/whitehall/blob/a656267d0bb9fb3e1ccd240cc02d7d64a2e5a2d2/app/models/image_data.rb#L41) so I think it's the direction of travel we want to go in.

> Alternatively, keeping the `AttachmentUploader.versions.keys.push(:original)` logic here future-proofs us from when we might reintroduce a variant to that uploader.

True, but also feels like YAGNI to me. We can always bring the code back in future if we need variants again.